cmd/charm/charmcmd/push_test.go

```diff
@@ -225,20 +223,13 @@ func (s *pushSuite) TestUploadCharm(c *gc.C) {
 func (s *pushSuite) TestUploadCharmNoIdFromRelativeDir(c *gc.C) {
 	repo := entitytesting.Repo
 	charmDir := filepath.Join(repo.Path(), "quantal/multi-series")
-	curDir, err := os.Getwd()
```

Comment:
Here we were testing real cases in which the user is placed on these directories. I guess this test should keep working as it was.